docs/docs/Get-Started/get-started-installation.md

```diff
@@ -3,170 +3,222 @@ title: Install Langflow
 slug: /get-started-installation
 ---
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 Langflow can be installed in two ways:
 
-* As a [Python package](#install-langflow-locally)
-* As a [standalone desktop application](#install-langflow-desktop)
+* As a [Python package](#install-and-run-langflow-oss) with Langflow OSS
+* As a [standalone desktop application](#install-langflow-desktop) with Langflow Desktop
```

Comment:
Have we been told to remove the references to DS LF installation?

Reply:
Checking

Reply:
Confirmed we keep it